src/main/java/seedu/address/MainApp.java

```diff
@@ -143,21 +146,23 @@ protected Config initConfig(Path configFilePath) {
      */
     protected UserPrefs initPrefs(UserPrefsStorage storage) {
         Path prefsFilePath = storage.getUserPrefsFilePath();
-        logger.info("Using prefs file : " + prefsFilePath);
+        logger.info("Using preference file : " + prefsFilePath);
 
         UserPrefs initializedPrefs;
         try {
             Optional<UserPrefs> prefsOptional = storage.readUserPrefs();
             if (!prefsOptional.isPresent()) {
-                logger.info("Prefs file not found. Will be starting with a new default prefs file.");
+                logger.info("Creating new preference file " + prefsFilePath);
             }
             initializedPrefs = prefsOptional.orElse(new UserPrefs());
         } catch (DataConversionException e) {
-            logger.warning("UserPrefs file at " + prefsFilePath + " is not in the correct format. "
-                    + "Using default user prefs");
+            logger.warning(
+                    "Preference file at " + prefsFilePath
+                            + " might be missing or corrupted. Using default preferences");
```

Comment:
Let's not do any fancy line wrappings. Wrap only when the line is close to the max line length allowed.